src/styles/_variables.sass

```diff
@@ -4,5 +4,6 @@ $primary-font: 'Montserrat', sans-serif
 // Colors
 $primary-color: #00c2ff
 $placeholder-color: #656565
+$background-color: #010101
```

Comment:
I have no context of what background color this is, try making it a bit clearer bit don't vaguely name it `$background-color`